devices/the_light_group.js

```diff
@@ -0,0 +1,27 @@
+const exposes = require('../lib/exposes');
+const fz = {...require('../converters/fromZigbee'), legacy: require('../lib/legacy').fromZigbee};
+const reporting = require('../lib/reporting');
+const extend = require('../lib/extend');
+const e = exposes.presets;
+
+module.exports = [
+    {
+        zigbeeModel: ['S57003'],
+        model: 'S57003',
+        vendor: 'The Light Group',
+        description: 'SLC SmartOne Zigbee wall remote 4-channels',
+        fromZigbee: [fz.command_on, fz.command_off, fz.battery, fz.command_move, fz.command_stop],
+        exposes: [e.battery(), e.action([
+            'on_l1', 'off_l1', 'brightness_move_up_l1', 'brightness_move_down_l1', 'brightness_stop_l1',
+            'on_l2', 'off_l2', 'brightness_move_up_l2', 'brightness_move_down_l2', 'brightness_stop_l2',
+            'on_l3', 'off_l3', 'brightness_move_up_l3', 'brightness_move_down_l3', 'brightness_stop_l3',
+            'on_l4', 'off_l4', 'brightness_move_up_l4', 'brightness_move_down_l4', 'brightness_stop_l4',
+        ])],
+        toZigbee: [],
+        meta: {multiEndpoint: true},
+        endpoint: (device) => {
+            return {l1: 1, l2: 2, l3: 3, l4: 4};
+        },
+        whiteLabel: [{vendor: 'Sunricher', model: 'SR-ZG9001K8-DIM'}],
```

Comment:
Have you checked that the surnicher variant uses the same `zigbeeModel` (`S57003`)? If not, I believe it will have a different `zigbeeModel` and thus does not have to be added as a whitelabel.

Reply:
No, I might be confused here and that I have misunderstood the intention of the whitelabel. 
I saw in the supported device overview that there are several versions of the 8 button wall switch (EcoDim. Namron, Robb, Sunricher) in addition to S57003. I believed that the Sunricher version was the "original" one and the other versions were white labels (but manufactured by Sunricher), and then I used the whitelabel based on what i found in robb.js ( zigbeeModel: ['ZG2833K4_EU06', 'ROB_200-008', 'ROB_200-008-0'],
). How is the whitelabel supposed to be used? Is it the other way around? 

But the zigbee model for "my device" is S57003. Here is the device info in the database.db:
`{"id":4,"type":"EndDevice","ieeeAddr":"0x5c0272fffefba4d8","nwkAddr":52432,"manufId":4644,"manufName":"THE LIGHT GROUP AS","powerSource":"Battery","modelId":"S57003","epList":[1,2,3,4],"endpoints":{"1":{"profId":260,"epId":1,"devId":1,"inClusterList":[0,1,3,2821,4096],"outClusterList":[3,4,5,6,8,25,768,4096],"clusters":{"genPowerCfg":{"attributes":{"batteryPercentageRemaining":100}},"genBasic":{"attributes":{"modelId":"S57003","manufacturerName":"THE LIGHT GROUP AS","powerSource":3,"zclVersion":3,"appVersion":0,"stackVersion":0,"hwVersion":0,"dateCode":"NULL","swBuildId":"2.7.6_r22"}}},"binds":[{"cluster":8,"type":"endpoint","deviceIeeeAddress":"0x680ae2fffe3a6593","endpointID":1}],"configuredReportings":[],"meta":{}},"2":{"profId":260,"epId":2,"devId":1,"inClusterList":[0,1,3,2821,4096],"outClusterList":[3,4,5,6,8,25,768,4096],"clusters":{"genBasic":{"attributes":{"appVersion":0,"stackVersion":0,"hwVersion":0}}},"binds":[],"configuredReportings":[],"meta":{}},"3":{"profId":260,"epId":3,"devId":1,"inClusterList":[0,1,3,2821,4096],"outClusterList":[3,4,5,6,8,25,768,4096],"clusters":{"genBasic":{"attributes":{"appVersion":0,"stackVersion":0,"hwVersion":0}}},"binds":[],"configuredReportings":[],"meta":{}},"4":{"profId":260,"epId":4,"devId":1,"inClusterList":[0,1,3,2821,4096],"outClusterList":[3,4,5,6,8,25,768,4096],"clusters":{"genBasic":{"attributes":{"appVersion":0,"stackVersion":0,"hwVersion":0}}},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":0,"stackVersion":0,"hwVersion":0,"dateCode":"NULL","swBuildId":"2.7.6_r22","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1633375258886}`

Reply:
Whitelabel should only be used when a different device is using the exact same `zigbeeModel` (meaning they cannot by distinguished from each other)

Reply:
thanks, I have deleted the whitelabel.